TestProject/cellularAutomat/Board.php

```diff
@@ -37,6 +37,11 @@ function __construct($_height, $_width)
         $this->height=$_height;
         $this->width=$_width;
 
+        /**
```

Comment:
Good comment! It could have been a bit shorter (one line) but better more documentation than less!
However, if it is an internal comment that should **NOT** be output in the generated documentation (and this is one) then don't start it with `/**` but instead use the regular `/*` to start it. (All doc-blocks that start with `/*` are ignored by documentation generation tools)